### PR TITLE
move add_blocks_in_batches() out of tests

### PR DESCRIPTION
**Pull request by arvidn**
since the node simulator depends on it.

This is what's causing CI to be red currently, the dependency from production code on tests.

### Purpose:

Fix CI, and the dependency cycle.

### CODE REVIEW

**Review by AmineKhaldi — APPROVED** [review]

**Review by Quexington — APPROVED** [review]

File: chia/_tests/util/misc.py [hunk 1]
```diff
@@ -638,49 +630,3 @@ def __ge__(self: T_ComparableEnum, other: T_ComparableEnum) -> object:
...
-        block_record = await full_node.blockchain.get_block_record_from_db(header_hash)
-        assert block_record is not None
-        ssi, diff = get_next_sub_slot_iters_and_difficulty(
-            full_node.constants, True, block_record, full_node.blockchain
-        )
-        fork_height = block_record.height
-        fork_info = ForkInfo(block_record.height, fork_height, block_record.header_hash)
-
-    _, dummy_node_id = await add_dummy_connection(full_node.server, "127.0.0.1", 12315)
-    dummy_peer = full_node.server.all_connections[dummy_node_id]
```
Comment (arvidn) [inline chia/_tests/util/misc.py:663]: note that I removed the `dummy_peer`, as that would have pulled in even more test code

Comment (Quexington) [reply chia/_tests/util/misc.py:663]: changes to test code -> tests passing == checks out to me

**Review by almogdepaz — APPROVED** [review]

**Review by wjblanke — APPROVED** [review]
aok

**Review by altendky — APPROVED** [review]
# :green_apple: :green_book: :green_circle: :green_heart: :pea_pod: :broccoli: :palm_tree: :tanabata_tree: :christmas_tree: :deciduous_tree: :evergreen_tree: :frog: :green_square: :green_salad: :pear:
```diff
+ *fingers crossed this is enough green to be merged*
```

**Comment by altendky** [comment]
```diff
@@ or maybe really it's the purple that is needed here to bring on the merging? @@
```

**Pull request merged**